### PR TITLE
Create local transaction schema

### PR DESCRIPTION
**Pull request by davidbasalla**
Add and generate local transaction schema.

https://trello.com/c/kfAEfcnM/639-migrate-localtransaction-format-to-rendered

### CODE REVIEW

File: formats/local_transaction/publisher/links.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
```
Comment (fofr) [inline formats/local_transaction/publisher/links.json]: This is empty, so the file can be omitted completely.

File: formats/local_transaction/publisher/details.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "service_tiers",
+    "introduction",
+    "more_information",
+    "need_to_know"
+  ],
+  "properties": {
+    "lgsl_code": {
+      "type": "integer"
+    },
+    "lgil_override": {
```
Comment (fofr) [inline formats/local_transaction/publisher/details.json]: Should this be a required field if the override can be null?

File: formats/local_transaction/publisher/details.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "anyOf": [
+        {
+          "type": "integer"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "service_tiers": {
```
Comment (fofr) [inline formats/local_transaction/publisher/details.json]: Could you include some descriptions for this field and `lgsl_code` and `lgil_override`?

**Comment by brenetic** [comment]
@fofr  Thanks for the review dude. Have made those changes and updated

**Pull request merged**